clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -1732,6 +1735,50 @@ public void testAutoCommitAfterCoordinatorBackToService() {
         assertEquals(100L, subscriptions.position(t1p).longValue());
     }
 
+    @Test
+    public void testAutoCommitAfterInvokingOffsetCommitCallback() throws InterruptedException {
+        final List<Long> committedOffset = new ArrayList<>();
+        ConsumerInterceptor ci = new ConsumerInterceptor<Object, Object>() {
```

Comment:
What's the advantage to doing this in an interceptor instead of the `OffsetCommitCallback`?

Reply:
The ut use interceptor to collect all successful offsets. The OffsetCommitCallback is invoked only once so it can't be substituted for interceptor.

Reply:
@hachikuji any comments about the tests? thanks.